analytics/views/support.py

```diff
@@ -209,12 +236,10 @@ def support(request: HttpRequest) -> HttpResponse:
                 context[
                     "success_message"
                 ] = f"{realm.string_id} downgraded and voided {voided_invoices_count} open invoices"
-        elif request.POST.get("scrub_realm", None) is not None:
-            if request.POST.get("scrub_realm") == "scrub_realm":
-                do_scrub_realm(realm, acting_user=acting_user)
-                context["success_message"] = f"{realm.string_id} scrubbed."
+        elif scrub_realm is not None:
```

Comment:
We defintely should not not be checking for `scrub_realm is not None`. Since value of scrub_realm can be `False` as well(even though it's not possible in the current UI). And `False is not None` evaluates to `True`. So if the API is called with "false" as the value of scrub_realm param it will result in the realm being scrubbed.

We should check for `scrub_realm is True` instead.

Pinging @timabbott to double check since this is a very sensitive param.

Reply:
We should be checking that `scrub_realm` is true to see if we want to scrub the realm.  Checking `is not None` would be correct were the goal to be checking if the parameter was sent by the client.